sympy/utilities/autowrap.py

```diff
@@ -572,43 +572,49 @@ def autowrap(expr, language=None, backend='f2py', tempdir=None, args=None,
         If True, autowrap will not mute the command line backends. This can be
         helpful for debugging.
     helpers : 3-tuple or iterable of 3-tuples, optional
-        Used to define auxiliary expressions needed for the main expr. If the
-        main expression needs to call a specialized function it should be
-        passed in via ``helpers``. Autowrap will then make sure that the
-        compiled main expression can link to the helper routine. Items should
-        be 3-tuples with (<function_name>, <sympy_expression>,
-        <argument_tuple>). It is mandatory to supply an argument sequence to
-        helper routines.
-    code_gen : CodeGen instance
-        An instance of a CodeGen subclass. Overrides ``language``.
-    include_dirs : [string]
-        A list of directories to search for C/C++ header files (in Unix form
-        for portability).
-    library_dirs : [string]
-        A list of directories to search for C/C++ libraries at link time.
-    libraries : [string]
-        A list of library names (not filenames or paths) to link against.
-    extra_compile_args : [string]
-        Any extra platform- and compiler-specific information to use when
-        compiling the source files in 'sources'.  For platforms and compilers
-        where "command line" makes sense, this is typically a list of
-        command-line arguments, but for other platforms it could be anything.
-    extra_link_args : [string]
-        Any extra platform- and compiler-specific information to use when
-        linking object files together to create the extension (or to create a
-        new static Python interpreter).  Similar interpretation as for
-        'extra_compile_args'.
```

Comment:
This seems that you have deleted much of the existing docstring. Was that intentional? Probably an accident.